### DISCUSSION TITLE
The scale attribute with min and max values

### DISCUSSION THREAD

**Discussion by Slarof** [PurpurMC/Purpur#1543] [Feature Requests]
Can you please add the ability to specify the minimum and maximum values for the scale attribute?

Have a nice and successful day

**Comment by granny** [comment]
Can you clarify what you mean with your request? What usecase would this have?

**Reply by Slarof** [reply]
Let's say horses have a max_health attribute setting that has a min and max features

I imagine it as, for example:
        attributes:
          scale:
            min: 0.1
            max: 16.0

**Reply by granny** [reply]
Horses (and variants that are similar, like Camels and Llamas) are the only mobs that specify a min/max value since they spawn (and create offspring) with a randomized attribute set for health, jump strength, and speed. Is your request to make scale work the same way on horses (and variants), or to make it randomized for *all* mobs?

**Reply by Slarof** [reply]
I want to be able to set a random size for all possible entities

Application examples:
- The player goes to the cave, and there are spiders of different sizes
- The player went to the jungle, and parrots live there, among which you can find small ones - replacing the missing baby parrots
